playground/app.js

```diff
@@ -386,7 +393,7 @@ class App extends Component {
     });
   };
 
-  setLiveValidate = ({ formData }) => this.setState({ liveValidate: formData });
+  setValidate = ({ formData }) => this.setState({ ...formData });
```

Comment:
Just call `setState(formData)` here. You may even want to try using `onChange={this.setState}` later.